front/lib/resources/agent_message_feedback_resource.ts

```diff
@@ -125,6 +145,44 @@ export class AgentMessageFeedbackResource extends BaseResource<AgentMessageFeedb
     return feedbacks;
   }
 
+  static async fetchConversationId(
+    agentMessageFeedbackId: string
+  ): Promise<string | null> {
+    const agentMessageFeedback = await AgentMessageFeedback.findByPk(
+      agentMessageFeedbackId,
+      {
+        // Feedback -> AgentMessage -> Message -> Conversation
+        include: [
+          {
+            model: AgentMessageModel,
+            attributes: ["id"],
+            include: [
+              {
+                model: Message,
+                as: "agentMessage",
```

Comment:
`as: "message"` no? 
It's a bit confusing to name is agentMessage when it's coming from the "Message" model.
Also little snake_case vs camelCase issue

Reply:
I tried a more friendly name but sequelize would complain because the alias is not what is expected (it seems to expect agentMessage specifically). Haven't looked for a workaround for more than 10 minutes though, so happy to jumb back to it if you think it's worth it :)